### ISSUE TITLE
Contribution agreement / results data license

### ISSUE THREAD

**Issue by lgarron** [dmint789/cubing-contests#1]
In order to let the website display results, and for others to download/use it, it would be good to have a policy that specifies how the data can be used, and by whom.

It would also be good to have a policy for matching WCA policies, e.g. around anonymizing a profile name on request.

**Comment by dmint789** [comment]
Yeah, I think this should definitely be added. Are you able to draft this?

**Comment by mintydev789** [comment]
This should be covered by the license section on the [exports page](https://cubingcontests.com/export) now
